{ 'stand_up': { 'left_arm': [ { 'duration': 1.0,
                                'name': u'stand_up_0',
                                'positions': [ -1.5878,
                                               1.2968,
                                               -0.0002,
                                               -0.0001,
                                               -0.7176,
                                               -0.2511,
                                               0.0001],
                                'starttime': 0.0},
                              { 'duration': 1.44,
                                'name': u'stand_up_1',
                                'positions': [ -2.0902,
                                               0.611,
                                               0.0024,
                                               -0.0012,
                                               -1.2463,
                                               0.0029,
                                               0.0023],
                                'starttime': 1.002},
                              { 'duration': 1.0,
                                'name': u'stand_up_1_2',
                                'positions': [ -2.2876,
                                               0.4934,
                                               -0.0042,
                                               -0.0285,
                                               -1.2588,
                                               -0.0098,
                                               0.0],
                                'starttime': 2.448},
                              { 'duration': 1.808,
                                'name': u'stand_up_2',
                                'positions': [ -0.268,
                                               -1.6445,
                                               -0.0204,
                                               -0.0428,
                                               -1.2599,
                                               -0.017,
                                               0.0001],
                                'starttime': 4.531},
                              { 'duration': 1.636,
                                'name': u'stand_up_3',
                                'positions': [ -0.3585,
                                               -2.1536,
                                               -0.0297,
                                               -0.0587,
                                               -1.2597,
                                               -0.03,
                                               0.0],
                                'starttime': 6.349},
                              { 'duration': 1.477,
                                'name': u'stand_up_4',
                                'positions': [ -0.8655,
                                               -1.8979,
                                               -0.0319,
                                               -0.0628,
                                               -1.2596,
                                               -0.0303,
                                               0.0001],
                                'starttime': 8.004},
                              { 'duration': 2.484,
                                'name': u'stand_up_5',
                                'positions': [ -0.0035,
                                               -1.917,
                                               -0.0177,
                                               -0.0688,
                                               -0.0209,
                                               -0.0387,
                                               0.001],
                                'starttime': 9.49},
                              { 'duration': 1.0,
                                'name': u'stand_up_6',
                                'positions': [ -0.01,
                                               -1.449,
                                               -0.0297,
                                               -0.07,
                                               -0.0299,
                                               -0.04,
                                               -0.0],
                                'starttime': 11.98},
                              { 'duration': 1.954,
                                'name': u'stand_up_7_0',
                                'positions': [ -0.2249,
                                               1.1116,
                                               -0.0208,
                                               -0.0697,
                                               -0.0309,
                                               -0.04,
                                               -0.0],
                                'starttime': 13.01}],
                'left_leg': [ { 'duration': 1.0,
                                'name': u'stand_up_0',
                                'positions': [ -0.0153,
                                               0.0076,
                                               -0.3478,
                                               0.7871,
                                               0.8288,
                                               -0.0573],
                                'starttime': 0.0},
                              { 'duration': 1.0,
                                'name': u'stand_up_1',
                                'positions': [ -0.0099,
                                               -0.0042,
                                               0.0158,
                                               0.5839,
                                               1.571,
                                               -0.0598],
                                'starttime': 3.498},
                              { 'duration': 1.812,
                                'name': u'stand_up_2',
                                'positions': [ -0.0104,
                                               -0.0104,
                                               -1.3423,
                                               2.1665,
                                               1.57,
                                               -0.06],
                                'starttime': 4.532},
                              { 'duration': 1.636,
                                'name': u'stand_up_3',
                                'positions': [ -0.0099,
                                               -0.0099,
                                               -1.5709,
                                               2.8879,
                                               1.5701,
                                               -0.06],
                                'starttime': 6.362},
                              { 'duration': 3.014,
                                'name': u'stand_up_8_t',
                                'positions': [ -0.0184,
                                               0.0049,
                                               -1.2165,
                                               3.1386,
                                               1.8122,
                                               -0.0317],
                                'starttime': 16.986}],
                'neck': [ { 'duration': 1.0,
                            'name': u'stand_up_0',
                            'positions': [0.0001, -0.9375],
                            'starttime': 0.0}],
                'right_arm': [ { 'duration': 1.0,
                                 'name': u'stand_up_0',
                                 'positions': [ 1.5878,
                                                -1.2968,
                                                0.0002,
                                                0.0001,
                                                0.7176,
                                                0.2511,
                                                -0.0001],
                                 'starttime': 0.0},
                               { 'duration': 1.4,
                                 'name': u'stand_up_1',
                                 'positions': [ 2.0902,
                                                -0.611,
                                                -0.0024,
                                                0.0012,
                                                1.2463,
                                                -0.0029,
                                                -0.0023],
                                 'starttime': 1.002},
                               { 'duration': 1.0,
                                 'name': u'stand_up_1_2',
                                 'positions': [ 2.2876,
                                                -0.4934,
                                                0.0042,
                                                0.0285,
                                                1.2588,
                                                0.0098,
                                                -0.0],
                                 'starttime': 2.417},
                               { 'duration': 1.812,
                                 'name': u'stand_up_2',
                                 'positions': [ 0.268,
                                                1.6445,
                                                0.0204,
                                                0.0428,
                                                1.2599,
                                                0.017,
                                                -0.0001],
                                 'starttime': 4.544},
                               { 'duration': 1.636,
                                 'name': u'stand_up_3',
                                 'positions': [ 0.3585,
                                                2.1536,
                                                0.0297,
                                                0.0587,
                                                1.2597,
                                                0.03,
                                                -0.0],
                                 'starttime': 6.362},
                               { 'duration': 1.477,
                                 'name': u'stand_up_4',
                                 'positions': [ 0.8655,
                                                1.8979,
                                                0.0319,
                                                0.0628,
                                                1.2596,
                                                0.0303,
                                                -0.0001],
                                 'starttime': 8.017},
                               { 'duration': 2.484,
                                 'name': u'stand_up_5',
                                 'positions': [ 0.0035,
                                                1.917,
                                                0.0177,
                                                0.0688,
                                                0.0209,
                                                0.0387,
                                                -0.001],
                                 'starttime': 9.522},
                               { 'duration': 1.0,
                                 'name': u'stand_up_6',
                                 'positions': [ 0.01,
                                                1.449,
                                                0.0297,
                                                0.07,
                                                0.0299,
                                                0.04,
                                                0.0],
                                 'starttime': 12.006},
                               { 'duration': 1.9949999999999992,
                                 'name': u'stand_up_7_0',
                                 'positions': [ 0.2249,
                                                -1.1116,
                                                0.0208,
                                                0.0697,
                                                0.0309,
                                                0.04,
                                                0.0],
                                 'starttime': 15.017000000000001}],
                'right_leg': [ { 'duration': 1.0,
                                 'name': u'stand_up_0',
                                 'positions': [ 0.0153,
                                                -0.0076,
                                                0.3478,
                                                -0.7871,
                                                -0.8288,
                                                0.0573],
                                 'starttime': 0.0},
                               { 'duration': 1.0,
                                 'name': u'stand_up_1',
                                 'positions': [ 0.0099,
                                                0.0042,
                                                -0.0158,
                                                -0.5839,
                                                -1.571,
                                                0.0598],
                                 'starttime': 3.498},
                               { 'duration': 1.812,
                                 'name': u'stand_up_2',
                                 'positions': [ 0.0104,
                                                0.0104,
                                                1.3423,
                                                -2.1665,
                                                -1.57,
                                                0.06],
                                 'starttime': 4.544},
                               { 'duration': 1.636,
                                 'name': u'stand_up_3',
                                 'positions': [ 0.0099,
                                                0.0099,
                                                1.5709,
                                                -2.8879,
                                                -1.5701,
                                                0.06],
                                 'starttime': 6.375},
                               { 'duration': 3.014,
                                 'name': u'stand_up_8_t',
                                 'positions': [ 0.0184,
                                                -0.0049,
                                                1.2165,
                                                -3.1386,
                                                -1.8122,
                                                0.0317],
                                 'starttime': 17.039}],
                'torso': [ { 'duration': 1.0,
                             'name': u'stand_up_0',
                             'positions': [0.0001, 0.0107],
                             'starttime': 0.0},
                           { 'duration': 1.84,
                             'name': u'stand_up_2',
                             'positions': [0.0016, 0.532],
                             'starttime': 4.544},
                           { 'duration': 1.636,
                             'name': u'stand_up_3',
                             'positions': [0.0002, 0.9397],
                             'starttime': 6.401},
                           { 'duration': 2.484,
                             'name': u'stand_up_5',
                             'positions': [0.0038, 1.2866],
                             'starttime': 9.517},
                           { 'duration': 1.0,
                             'name': u'stand_up_6',
                             'positions': [-0.0003, 1.0394],
                             'starttime': 12.006},
                           { 'duration': 2.007,
                             'name': u'stand_up_7_0',
                             'positions': [-0.0002, -0.6233],
                             'starttime': 15.018},
                           { 'duration': 3.014,
                             'name': u'stand_up_8_t',
                             'positions': [0.018, 0.4743],
                             'starttime': 17.039}]}}
